app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1317,7 +1317,7 @@ public void updateMarker(final boolean isBookmarked, final Place place, @Nullabl
             return (isBookmarked ?
                 R.drawable.ic_custom_map_marker_green_bookmarked :
                 R.drawable.ic_custom_map_marker_green);
-        } else if (!place.destroyed.trim().isEmpty()) { // Means place is destroyed
+        } else if (!place.exists) { // Means place is destroyed
```

Comment:
Please reword the comment to `// Means that the topic of the Wikidata item does not exist in the real world anymore, for instance it is a past event, or a place that was destroyed.`
Thanks!

Reply:
Added!